commit 696e256fd9236f155e30c636f70556239eb527bd (refs/changes/21/4421/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-22T18:59:32-07:00 (1 year, 4 months ago)
    
    MANTA-3787 waferlock should look for "zookeeper" as well as "binder"

diff --git a/sapi_manifests/waferlock/template b/sapi_manifests/waferlock/template
index 3b61b52..7609bf9 100644
--- a/sapi_manifests/waferlock/template
+++ b/sapi_manifests/waferlock/template
@@ -26,7 +26,9 @@
 			"{{{moray_domain}}}",
 			"{{{BINDER_SERVICE}}}",
 			{{! Don't use pgstatsmon_domain, might not be set. }}
-			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}"
+			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}",
+			{{! Same deal here: this is for legacy DCs. }}
+			"zookeeper.{{{datacenter_name}}}.{{{dns_domain}}}"
 		],
 		"sapi_url": "{{{sapi-url}}}",
 		"sapi_services": ["sdc/moray"],
